src/SnackBar/SnackBar.scss

```diff
@@ -0,0 +1,94 @@
+
+.snackbar-menu {
+    --snackbar-background-color: rgb(51, 51, 51);
+    --snackbar-background-color-hover: rgb(84, 84, 84);
+    --snackbar-text-label-color: rgba(255, 255, 255, 0.87);
+    --snackbar-text-action-color: rgb(187, 134, 252);
+    --snackbar-text-action-disabled-color: rgb(140, 140, 140);
+    z-index : 8;
+    margin : 0.5rem;
+    position : fixed;
+    right : 0;
```

Comment:
Probably makes sense to show it to the left of the screen. Or to make this configurable.